common/app/views/mainAMP.scala.html

```diff
@@ -40,6 +42,22 @@
 
             @body
 
+            <div class="fc-container__inner">
+                <div class="fc-container__header__title">
+                    <span>popular</span>
+                </div>
+                <amp-list width="600" height="1000" layout="responsive" src="@if(Play.isDev) {https://www.thegulocal.com} else {@Configuration.ajax.url}/most-read-mf2.json">
```

Comment:
ah now I realise - it actually checks the first 6 characters are https: - I raised a similar issue against amp a while ago and they fixed it (for data: urls in iframes) so they might fix it here too